fullstop-violation/fullstop-violation-jpa/src/generated/queryDsl/org/zalando/stups/fullstop/violation/domain/QAbstractCreatableEntity.java

```diff
@@ -0,0 +1,44 @@
+package org.zalando.stups.fullstop.violation.domain;
+
+import static com.mysema.query.types.PathMetadataFactory.*;
+
+import com.mysema.query.types.path.*;
+
+import com.mysema.query.types.PathMetadata;
+import javax.annotation.Generated;
+import com.mysema.query.types.Path;
+
+
+/**
+ * QAbstractCreatableEntity is a Querydsl query type for AbstractCreatableEntity
+ */
+@Generated("com.mysema.query.codegen.SupertypeSerializer")
+public class QAbstractCreatableEntity extends EntityPathBase<AbstractCreatableEntity> {
+
```

Comment:
yes, at the beginning I did that to simplify the configuration for the IDE. But I also change that time ago. Maybe was a problem with merging or refactoring. 
So you can change it, for me.